### PR TITLE
#3 #162966290 Adds a registration page 

### PR DESCRIPTION
**Pull request by jackiegitari1234**
**What does this PR do?**
- creates a user registration page

**Description of Task to be completed**
-  create a html registration page
- Create a form with well labeled input fields and a submit button 

**How should this be manually tested?**
- clone the repository `https://github.com/jackiegitari1234/Questioner` to your local machine
- navigate into Questioner/UI by executing the command `cd Questioner/UI`
- open the file index.html using a browser
- click the `Register` or `Join Us Today` link

**What are the relevant pivotal tracker stories?**
`https://www.pivotaltracker.com/n/projects/2235314`
#162966290

**Screenshot**
regi

### CODE REVIEW

**Comment by Ogollah** [comment]
Nice work here.

**Comment by wachiranduati** [comment]
good to go.... :100:

File: UI/register.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            </form>
+            
+        </div>
+
+        <div class="footer">
+            Copyright©2019
+        </div>
+
+    </body>
+</html>
```
Comment (Ogollah) [inline UI/register.html]: Add a new line

**Comment by colMike** [comment]
Really good work..However, what about the use of a max of 4 colours on a page?

**Comment by SolomonMacharia** [comment]
Nice UI. You may also consider changing the button to read 'Sign up' or 'Register', but there's still nothing wrong with submit. Good work though.

**Comment by Ogutu-Brian** [comment]
Nice work @jackiegitari1234

File: UI/register.html (hunk 1 above)
Comment (jackiegitari1234) [reply UI/register.html]: thanks for that

**Comment by jackiegitari1234** [comment]
thank you all, @SolomonMacharia  and @colMike am fixing that

**Pull request merged**